docs/development.md

```diff
@@ -162,6 +162,13 @@ $ ./rootfs/etc/nginx/lua/test/up.sh
 $ make lua-test
 ```
 
+To run tests in a pre-configured docker environment, run:
```

Comment:
Please remove this change. This is the default when you run `make test` or `make lua-test`

Reply:
Done, thanks!